llama_hub/minio/minio-client/base.py

```diff
@@ -12,6 +12,7 @@
 from llama_index.readers.base import BaseReader
 from llama_index.readers.schema.base import Document
 
+import urllib3
```

Comment:
Move this import inside `load_data` method. We follow importing external libraries inside the class to avoid any build failure